packages/plugins/script/write-initcode.ts

```diff
@@ -0,0 +1,55 @@
+import { ethers } from 'ethers'
+import { getSphinxConstants, remove0x } from '@sphinx-labs/contracts'
+
+/**
+ * Writes the initcode and expected addresses of all Sphinx system contracts. This allows us to use
+ * the contracts in Forge tests.
```

Comment:
Nit: This is not directly related to this PR, but it would be nice to document why we do not need to do this in the main library (we deploy the core contracts on local networks using `ensureSphinxAndGnosisSafeDeployed`).